### PR TITLE
stress test playground cluster buildout

### PR DESCRIPTION
**Pull request by ckairen**
Migration to chaos mesh 2.1.5 + westus2->westus3

### CODE REVIEW

File: tools/stress-cluster/cluster/azure/parameters/pg.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    "staticTestSecretsKeyvaultGroup": {
+      "value": "rg-stress-secrets-pg"
+    },
+    "enableHighMemAgentPool": {
+        "value": true
+    },
+    "tags": {
+      "value": {
+        "environment": "Pg",
```
Comment (benbp) [inline tools/stress-cluster/cluster/azure/parameters/pg.json:28]: Let's keep this lowercase.

File: tools/stress-cluster/cluster/kubernetes/stress-test-addons/values.yaml [hunk 2]
```diff
@@ -1,46 +1,46 @@
 env: test
 appInsightsKeySecretName:
-  test: appInsightsInstrumentationKey-tbiruti6oi24k
+  pg: appInsightsInstrumentationKey-s7b6dif73rup6
```
Comment (benbp) [inline tools/stress-cluster/cluster/kubernetes/stress-test-addons/values.yaml:3]: If you're going to replace test in this PR, then let's also remove `test.json` in this PR.

File: tools/stress-cluster/cluster/README.md [hunk 3]
```diff
@@ -78,13 +78,13 @@ Avoid checking in the updated dev values, they are for local use only.
...
-## Test Cluster
+## Playground Cluster
 
-The test cluster is the main ad-hoc cluster made available to SDK developers and partners. Changes to this cluster
+The playground cluster is the main ad-hoc cluster made available to SDK developers and partners. Changes to this cluster
 should be made carefully and announced in advance in order not to disrupt people's work.
 
 ```
-./provision.ps1 -env test
+./provision.ps1 -env pg
```
Comment (benbp) [inline tools/stress-cluster/cluster/README.md:87]: Provision script also needs a change here.

Comment (ckairen) [reply tools/stress-cluster/cluster/README.md:87]: What do you mean by provision script change. The provision script takes in $Environment parameter and just use those, I don't think there's any hard coded "test cluster" things.

**Comment by benbp** [comment]
Somehow [this](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/cluster/azure/monitoring/stress-status-workbook.bicep#L182) ended up being hardcoded. I probably missed it in my initial commit. I think it can either be removed or replaced.

**Comment by benbp** [comment]
LGTM for provision related changes. As we talked about the doc and script updates on the user side will be a follow-on update.

**Review by benbp — APPROVED** [review]

File: tools/stress-cluster/cluster/README.md (hunk 3 above)
Comment (ckairen) [reply tools/stress-cluster/cluster/README.md:87]: Followup in #3491

**Pull request merged**